controllers/nodepools.go

```diff
@@ -0,0 +1,139 @@
+package controllers
+
+import (
+	"context"
+	"strings"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	arbv1 "github.com/project-codeflare/multi-cluster-app-dispatcher/pkg/apis/controller/v1beta1"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+func (r *AppWrapperReconciler) scaleNodePool(ctx context.Context, aw *arbv1.AppWrapper, demandPerInstanceType map[string]int) (ctrl.Result, error) {
+	logger := ctrl.LoggerFrom(ctx)
+	connection, err := r.createOCMConnection()
```

Comment:
It looks like a new OCM connection is provisioned every time InstaScale needs to communicate with OCM.
Wouldn't it have more sense to reuse connection?